package.json

```diff
@@ -423,6 +424,11 @@
             "none"
           ]
         },
+        "puppet.titleBar.pdkNewModule.enable": {
+          "type": "boolean",
+          "default": false,
```

Comment:
Shouldn't this be `true` to keep existing functionality?

Reply:
Yes, it should. I had it default to test how it looked disabled and enabled, I missed changing it back.